lumigator_demo.py

```diff
@@ -10,12 +10,11 @@
 import pandas as pd
 import requests
 
-# URL where the API can be reached
+# APP URL
 API_HOST = os.environ["LUMIGATOR_SERVICE_HOST"]
-API_URL = f"http://{API_HOST}/api/v1"
+API_URL = f"https://{API_HOST}/api/v1"
```

Comment:
💯

Reply:
Could we do this as a follow on? would like to get this merged and iterate

Reply:
@veekaybee my LUMIGATOR_SERVICE_HOST is not the human-friendly URL but this ip: 10.100.7.142 which did not like HTTPs:
<img width="1091" alt="image" src="https://github.com/user-attachments/assets/a42deff7-b4f6-4a06-afcc-b08fce573def">

@chainlink if you clone the repo manually you can then checkout to this branch and directly test the code without the need to merge broken stuff.

Reply:
Btw I restarted the notebook server, tried this and it broke again. 
I set `API_URL = f"https://lumigator.mzai.dev/api/v1"` and it works.
So for the time being, we can either update `lumigator_demo.py` to set `API_URL` that way or update the ENV variable accordingly on AWS